lib/github.atd

```diff
@@ -590,7 +600,7 @@ type repo_issue_event = {
   ?label: base_label option;
   ?assignee: linked_user option;
   ?assigner: linked_user option;
-  ?milestone: milestone option;
+  ?milestone: repo_issue_milestone option;
```

Comment:
Is this change part of the feature? What is the purpose of it? Are repo issue events with milestones really using a totally different milestone type?